packages/keycloak-user-management/src/components/UserList/TableActions/index.tsx

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable @typescript-eslint/camelcase */
 import * as React from 'react';
 import { Popconfirm, Space } from 'antd';
 import { deleteUser } from './utils';
 import { Link } from 'react-router-dom';
 import { removeKeycloakUsers, KeycloakUser } from '../../../ducks/user';
 import { URL_USER_EDIT } from '../../../constants';
+import { Dictionary } from '@onaio/utils/dist/types/types';
```

Comment:
Should be `import { Dictionary } from '@onaio/utils';` Otherwise based on past experience the app will crash